crates/nu-command/src/filesystem/rm.rs

```diff
@@ -318,8 +318,8 @@ fn rm(
     }
 
     all_targets
-        .into_keys()
-        .map(move |f| {
```

Comment:
In case it's not clear: This is shadowing the existing `span` variable for each of the errors that may happen here. This is desired, because all failures from this point are target-specific.